### DISCUSSION TITLE
［求助］恐怖素材收集，胆大来

### DISCUSSION THREAD

**Discussion by schalesensei843** [share121/inter-knot#2955] [其他]
前往被空洞吞没的双子楼，验证流传的恐怖事迹，收集小说灵感，价格可谈，要胆大的

**Comment by 2ningmeng** [comment]
这个委托我接了<img src="https://upload-bbs.miyoushe.com/upload/2023/01/18/8481df2cd99d2b00ecb94a92a14c9fe4_7915718047845434249.png" alt="米游兔 OK" width="75" >
